pyerrors/fits.py

```diff
@@ -406,14 +418,19 @@ def chisqfunc(p):
     if not m.fmin.is_valid:
         raise Exception('The minimization procedure did not converge.')
 
-    hess_inv = np.linalg.pinv(jacobian(jacobian(chisqfunc))(params))
+    hess = jacobian(jacobian(chisqfunc))(params)
```

Comment:
You could use the `hessian()` here as well, right?

Reply:
Yes, for now I only adapted the `_standard_fit` function, but I can do that in the next iteration.